packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -58,12 +58,12 @@ const ESGlobalKeys = new Set([
     'Error', // Unstable
     'EvalError',
     'FinalizationRegistry',
-    'Float32Array',
-    'Float64Array',
+    // 'Float32Array', // Remapped
+    // 'Float64Array', // Remapped
     'Function', // dangerous
-    'Int8Array',
-    'Int16Array',
-    'Int32Array',
+    // 'Int8Array', // Remapped
```

Comment:
they should be here.... since they are language global names... but they probably should be in `ReflectiveIntrinsicObjectNames`, which I expect to cause a throw error in the test that you have.

Reply:
That's what I reported to you in Slack. I initially moved them to `ReflectiveIntrinsicObjectNames` and was able to observe the thrown exceptions. When I removed them from both lists, the failure goes away. Removal from this list fits the "remap" case... But as I asked also asked in slack: what is cost? what are the consequences?

Reply:
I've tossed this out.